src/Aspire.ProjectTemplates/templates/aspire-starter/Aspire-StarterApplication.1.Tests/Aspire-StarterApplication.1.Tests.csproj

```diff
@@ -30,10 +30,14 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Using Include="System.Net" />
```

Comment:
What's this for?

Reply:
`HttpStatusCode.OK` in the test. Not sure why it wasn't there before, I'll recheck.

Reply:
Ah adding it meant I could remove all the `using` statements from the test file. Given app hosts will often be using `HttpClient` and asserting against related types it felt worthwhile.